README.md

```diff
@@ -6,17 +6,24 @@ This action allows you to automate the release process of your npm modules. It c
 
 ### What does it do?
 
-- When run, it opens a new PR for the release.
+- When run, it opens a new PR for the release creating a new branch named `release/${new semver version}`.
 - When/if the PR gets merged, it publishes a new Npm release and a new Github release with change logs
 
 You can also use it for releases without Npm. In that case, when the PR merges, a new GitHub release will be published. Which you can use to trigger another workflow that deploys the app somewhere (GCP, AWS etc).
 
 ### Usage
 
-- Install the [optic-release-automation](https://github.com/apps/optic-release-automation) GitHub app to your organization (or selected repositories)
+- _(Optional)_ Install the [optic-release-automation](https://github.com/apps/optic-release-automation) GitHub app to your organization (or selected repositories)
 - Create a new workflow file at `.github/workflows/release.yml` (from example below) with one step that uses this action and supply the inputs.
 
-#### Example
+Note that the `on` triggers are mandatory:
+
+- `workflow_dispatch`: to start the new release process
+- `pull_request`: to complete the release process when the PR is merged
```

Comment:
since you're specifying this, you may want to specify that the only state we need is `closed`.